ui/StatusQ/src/onboarding/enums.h

```diff
@@ -66,6 +66,7 @@ class OnboardingEnums: public QObject
         MaxPairingSlotsReached,
         BlockedPIN, // PIN remaining attempts == 0
         BlockedPUK, // PUK remaining attempts == 0
+        FactoryResetting,
```

Comment:
If merged after https://github.com/status-im/status-desktop/pull/17127, please add it to here as well:
https://github.com/status-im/status-desktop/blob/6dacf2f50d6f431b767d21ff19cd9e95e4993021/src/app_service/service/keycardV2/dto.nim#L24

Reply:
Alright, I can wait, will add this state

Reply:
@igor-sirotin what about the missing `startKeycardFactoryReset()` NIM method?

Reply:
Alright, I added the state in my PR: 2a25314d1be81b34a21457c32716e95b53743447

Reply:
@caybro What's the `startKeycardFactoryReset`, I have no idea about it

Reply:
This should invoke the actual factory reset method on the backend side, via the respective store function